### DISCUSSION TITLE
Multiple n-n relations to the same table

### DISCUSSION THREAD

**Discussion by IRediTOTO** [prisma/prisma#20418] [Q&A]
Hi I have tables like this
```prisma
model Game {
  id          Int      @id @default(autoincrement())
...
  Thumb        Image[] @relation("ThumbGame")
  ScreenShots Image[] @relation("ScreenshotGome")

}

model Image {
  id             String            @id
  slug          String            @unique
  ...
  GameScreenshot @relation("ScreenshotGome")
  GameThumb      Game[] @relation("ThumbGame")

}
```

Everything is fine when run `npx prisma db push`  But in the code when I can't create many records of ThumbGame like this

```js
await prm.game.create({
            data: {
               // ... 
                
                ScreenShots: {
                    createMany: gallery.map((i) => ({  // Array of object here but there is no createMany in the 
                        id: i.id,
                        slug: i.publicId,
                    })),
                },
            },
        });
```
![image](https://github.com/prisma/prisma/assets/15695252/aaf3a2ed-202a-42f7-9c22-6911222a846d)
I want to create many image here, but there is no create many. how can I solve this?

**Comment by nurul3101** [comment]
Hello @IRediTOTO 👋 

`createMany` is not supported in nested writes for relational data in Prisma. This is why you're not able to use `createMany` for the ScreenShots relation in your example.

We have a feature request for it here:
- https://github.com/prisma/prisma/issues/5455

It would be great if you can add a 👍  to the feature request, it would help our product team in prioritising it.

For now, you can use create in a loop to create multiple records, it would look something like this:
```ts
for (const image of gallery) {
    await prisma.game.create({
        data: {
            // ...
            ScreenShots: {
                create: {
                    id: image.id,
                    slug: image.publicId,
                },
            },
        },
    });
}
```

It would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 
Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Reply by nurul3101** [reply]
Just checking to see if you had a chance to look at my last response.
Please let us know in case you still need assistance. If not, please consider marking the discussion as answered. 🙏

**Reply by nurul3101** [reply]
I'll mark this discussion as resolved.
Please feel free to open a new discussion if needed.
